.github/workflows/pr.yml

```diff
@@ -29,5 +29,5 @@ jobs:
       - name: npm test
         run: |
           echo ----RUNNING TESTS-----
-          npm run test-once
-          echo ----RUNNING TESTS----
+          npm run dev & npm run test-once
```

Comment:
single & allows dev to run along side run test while double & has to wait for dev to complete which it does not.